packages/shorebird_cli/lib/src/shorebird_artifacts.dart

```diff
@@ -71,6 +81,20 @@ class ShorebirdCachedArtifacts implements ShorebirdArtifacts {
     );
   }
 
+  File get _analyzeSnapshotMacosFile {
+    return File(
+      p.join(
+        shorebirdEnv.flutterDirectory.path,
+        'bin',
+        'cache',
+        'artifacts',
+        'engine',
+        'darwin-x64-release',
```

Comment:
We're using the enum to differentiate based on Target, but we may also need to differentiate based on Host

Reply:
Yep, we'll probably want to refactor `ShorebirdArtifacts` to accept host and/or target in the future.